developers_chamber/qa/checks.py

```diff
@@ -137,7 +137,7 @@ def _run_check(self):
             if diff_obj.b_path and self._is_python_file(diff_obj.b_path):
                 for line in diff_obj.diff.decode().split('\n'):
                     if line.startswith('+'):
-                        match = re.search(self.pattern, line)
+                        match = re.search(self.pattern, line[1:])
```

Comment:
Not really needed for this fix, but the "+" sign is really not part of the code.